src/view.rs

```diff
@@ -114,8 +182,8 @@ pub fn update_initial_window_state(ui: &mut Ui, state: &mut State, ids: &mut Ids
     // FIXME LinuCC For some reason `ui.kids_bounding_box()` accesses the
     //     `ui.prev_updated_widgets`, which only exists after generating the Ui
     //     a second time.
-    set_ui(ui.set_widgets(), &state, &state.selected_command, ids);
-    set_ui(ui.set_widgets(), &state, &state.selected_command, ids);
+    set_ui(ui.set_widgets(), state, ids);
+    set_ui(ui.set_widgets(), state, ids);
```

Comment:
o_O

Reply:
O_o

Reply:
Ah, I guess you mean not passing the state by reference? Whoops

Reply:
actually I mostly mean the double call...

Reply:
Yeah thats a WTF, but their internal data structure doesn't expect me to access `kids_bounding_box` this early on. Welp